extensions/script/ExecuteScript.cpp

```diff
@@ -123,6 +128,10 @@ void ExecuteScript::onTrigger(const std::shared_ptr<core::ProcessContext> &conte
     throw std::runtime_error("No script engine available");
   }
 
+  if (module_directory_.size()) {
```

Comment:
should we use `!module_directory_.empty()`?

(or better yet `std::optional<std::string>`, although I see that this member is not a new addition)

Reply:
Updated module directory to be optional in f6be6d51339473f54b18c103fdef812b47c759ee